1-js/09-classes/01-class/article.md

```diff
@@ -1,22 +1,22 @@
 
-# Class basic syntax
+# Sintaxă de bază a claselor
 
 ```quote author="Wikipedia"
-In object-oriented programming, a *class* is an extensible program-code-template for creating objects, providing initial values for state (member variables) and implementations of behavior (member functions or methods).
+In programarea orientată obiect, o clasă este un code template pentru crearea de obiecte, pentru a furniza valori inițiale variabileleor membre ale clasei și de a implementa comportamentul acesteia(funcții membre și metode).
```

Comment:
Missing translation : "...a class is an extensible program..." -> "o clasa este un program extensibil"